media/css/foundation/foundation.less

```diff
@@ -2,258 +2,426 @@
+        }
+
+        &:before {
+            background: @mozillaRed url('/media/img/newsletter/mozorg-newsletter.svg') center center no-repeat;
+            .background-size(100px, 100px);
+            border-radius: 100%;
+            content: '';
+            display: block;
+            height: 150px;
+            left: 0;
+            position: absolute;
+            top: 0;
+            width: 150px;
+        }
+    }
 
-  #masthead,
-  #main-content,
-  .billboard {
-    width: @widthTablet - (@gridGutterWidth * 2);
-  }
+    a:link,
+    a:visited {
+        color: @linkBlue;
+    }
 
-  #main-content {
-    .intro {
-      .span_narrow(4);
-      .font-size(22px);
+    a:hover,
+    a:focus {
+        color: @linkBlueHover;
     }
 
-    ul.links {
-      .span_narrow(4);
-      h4 {
-        .font-size(20px);
-      }
+    input[type='email'] {
+        border: 1px solid @borderColor;
     }
 
-  }
+    #newsletter-form-thankyou {
+        color: @textColorPrimary;
 
+        h3 {
+            color: @textColorPrimary;
+        }
+    }
 }
 
-/* Mobile layout: 320px */
-@media only screen and (max-width: @breakTablet) {
 
-  #masthead,
-  #main-content,
-  .billboard {
-    width: @widthMobile - (@gridGutterWidth * 2);
-  }
 
-  .billboard {
-    padding-left: @gridGutterWidth;
-    padding-right: @gridGutterWidth;
-  }
-
-  #main-content {
-    margin-bottom: @baseLine;
+/* Tablet Layout: 760px */
+@media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
 
-    .title-banner {
-      width: 240px;
-      .font-size(28px);
-      letter-spacing: -.5px;
-      margin: -25px 0 @baseLine;
+    #masthead,
+    .content,
+    .billboard {
+        width: @widthTablet - (@gridGutterWidth * 2);
     }
 
-    .title-banner:after {
-      display: none;
+    .page-header {
+        background-image: url('/media/img/foundation/banner/foundation_hero-image-medium.jpg');
     }
 
-    .intro {
-      width: auto;
-      .font-size(1em);
-      margin-bottom: @baseLine;
+    .content-main {
+        .main-column {
+            .span_narrow(8);
+        }
+
+        .sidebar {
+            .offset_narrow(1);
+        }
+
+        .video {
+            .span_narrow(6);
+            margin-bottom: @baseLine;
+        }
+
+        .intro {
+            .span_narrow(6);
+            .font-size(22px);
+        }
+
+        ul.links {
+            .span_narrow(6);
+            h4 {
+                .font-size(20px);
+            }
+        }
+
     }
 
-    ul.links {
-      width: auto;
-      margin-bottom: 0;
-      li {
-        min-height: 0;
-      }
+    #newsletter-subscribe .form-title {
+        h3 {
+            .font-size(24px);
+        }
+
+        h4 {
+            .font-size(16px);
+        }
+
+        &:before {
+            width: 120px;
+            height: 120px;
+            .background-size(80px, 80px);
+        }
     }
 
-  }
+    html[dir="rtl"] {
+        #main-content .title-shadow-box {
```

Comment:
This redesign gets rid of the title shadow box so we don't need this block of CSS.